src/sentry/integrations/jira/notify_action.py

```diff
@@ -1,39 +1,29 @@
 from __future__ import absolute_import
 
 import logging
-import six
 
 from django import forms
 from django.utils.translation import ugettext_lazy as _
 
 from sentry.integrations.jira.utils import (
     transform_jira_fields_to_form_fields,
     transform_jira_choices_to_strings,
-    get_name_for_jira,
 )
 from sentry.models.integration import Integration
-from sentry.rules.actions.base import TicketEventAction
+from sentry.rules.actions.base import TicketEventAction, NotifyServiceForm, INTEGRATION_KEY
 from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.utils.http import absolute_uri
 from sentry.web.decorators import transaction_start
 
 
 logger = logging.getLogger("sentry.rules")
 
-INTEGRATION_KEY = "integration"
 
-
-class JiraNotifyServiceForm(forms.Form):
+class JiraNotifyServiceForm(NotifyServiceForm):
```

Comment:
@ceorourke I don't think we even need to subclass `NotifyServiceForm`, can't we just use it directly if you move the `integration = forms.ChoiceField` line into the base class?